duradmin/src/main/webapp/WEB-INF/jsp/login-v2.jsp

```diff
@@ -16,40 +16,43 @@
 				evt.stopPropagation();
 				dc.login($("#loginForm"));
 			});
-		});		
+		});
 		</script>
 
 		<form id="loginForm" action="${pageContext.request.contextPath}/login"  method="post" onsubmit="return false;" >
 			<div id="login-wrapper">
 				<div id="login-header" class="outer clearfix">
-					<div id="dc-logo-panel"><a href="/duradmin/spaces" id="dc-logo"></a></div>			
+					<div id="dc-logo-panel"><a href="/duradmin/spaces" id="dc-logo"></a></div>
 				</div>
 				<div id="login-content" class="pane-L1-body clearfix">
 					<h1 id="title" class="float-l">Login</h1>
 					<div id="form-fields" class="form-fields float-r">
 						<div id="msg-error" class="error" style="display:none">Username/Password combination not valid. Please try again.</div>
-						
+
 						<ul>
 							<li class="clearfix">
-								<label for="username">Username</label>							
+								<label for="username">Username</label>
 								<input type="text" id="username" name="username" class="field"/>
 							</li>
 							<li class="clearfix">
 								<label for="password">Password</label>
 								<input id="password" type="password" name="password" class="field"/>
 							</li>
 							<li class="clearfix">
-								<button id="button-login" class="primary  float-r">Login</button>											
-								<span id="feedback" style="display:none; color:white" class="primary  float-r"><img src="/duradmin/images/wait.gif"/>Logging in...</span>											
+								<button id="button-login" class="primary  float-r">Login</button>
+								<span id="feedback" style="display:none; color:white" class="primary  float-r"><img src="/duradmin/images/wait.gif"/>Logging in...</span>
 
 							</li>
 						</ul>
-					
+						<ul class="horizontal-list">

```

Comment:
CSS updates should be added to make the display of the Forgot Password link more consistent with the MC, such as:
```
float: right;
text-decoration: none;
margin-top: 10px;
margin-right: 12px;
```